mappings/net/minecraft/client/render/entity/FoxEntityRenderer.mapping

```diff
@@ -2,4 +2,4 @@ CLASS dpq net/minecraft/client/render/entity/FoxEntityRenderer
 	FIELD a SKIN Lqs;
 	FIELD j SKIN_SLEEP Lqs;
 	FIELD k SKIN_SNOW Lqs;
-	FIELD l SKIN_SNOW_SLEEp Lqs;
+	FIELD l SKIN_SNOW_SLEEP Lqs;
```

Comment:
All these fields should be named in the fashion `SLEEP_SKIN`, etc. like class names, I think.

Reply:
There's incomplete consistency, there.

Reply:
fixed